README.md

```diff
@@ -4,6 +4,9 @@ Send and receive files over available connections. [FAQ](https://github.com/geno
 ## Get it on
 [<img src="https://f-droid.org/badge/get-it-on.png" width="230">](https://f-droid.org/packages/com.genonbeta.TrebleShot/) [<img src="assets/google-play-badge.png" width="230">](https://play.google.com/store/apps/details?id=com.genonbeta.TrebleShot)
```

Comment:
Could it better if it were under 'Get it on' header?

Reply:
seems batter?

Reply:
Yeah. Thanks :)